### PR TITLE
Add test for linear-gradient with transparent/rgba(0,0,0,0)

### PR DESCRIPTION
**Pull request by lukebjerring**
sRGBA should be precomputed for aesthetic reasons.
See https://www.w3.org/TR/css3-images/#color-stop-syntax for explanation.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18984)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/281416808)

File: css/css-images-3/gradients-with-transparent.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Gradients with transparent</title>
+<link rel="author" title="Luke Bjerring" href="mailto:lukebjerring@chromium.org">
```
Comment (foolip) [inline css/css-images-3/gradients-with-transparent.html]: FYI, this is optional, even if lots of files have it.

**Review by foolip — APPROVED** [review]
This is great. Do you know if there's a WebKit bug that you could update with this test once it's landed and seen to only be failing in Safari?

**Comment by foolip** [comment]
On the URL, can you use https://drafts.csswg.org/css-images-3/#color-stop-syntax instead? https://www.w3.org/TR/css3-images/ hasn't been updated in a long time, and a lot of the time TR is like that.

On the terminology, the spec seems to call it premultiplied.

File: .travis.yml [hunk 2]
```diff
@@ -100,4 +100,4 @@ notifications:
   email:
     on_success: never
     on_failure: always
-  webhooks: https://pulls.web-platform-tests.org/api/build
+  webhooks: https://pulls-staging.web-platform-tests.org/api/build
```
Comment (JKereliuk) [inline .travis.yml]: Are you sure this should be in this PR?

Comment (sideshowbarker) [reply .travis.yml]: -  webhooks: https://pulls.web-platform-tests.org/api/build
    +  webhooks: https://pulls-staging.web-platform-tests.org/api/build
> Are you sure this should be in this PR?

@lukebjerring Is there a reason you went ahead and merged the branch for this change to master without responding to this review comment?

Comment (lukebjerring) [reply .travis.yml]: No - accidentally done due to lack of familiarity with github, and unfortunately the comment wasn't flagged as needing action. Sorry for any inconvenience - I quickly reverted immediately afterward.

**Pull request merged**